### ISSUE TITLE
Switching Step #5 and Step #6

### ISSUE THREAD

**Issue by aberdean** [open-learning-exchange/open-learning-exchange.github.io#271]
It may be better to practice synching/merging, before starting working on issues.

**Comment by dogi** [comment]
@aberdean hmm explain more are there additional changes necessary - or is your only suggestion to switch both steps

**Comment by aberdean** [comment]
@dogi my suggestion is just to switch the two steps, since I find it more intuitive to familiarize oneself with the whole process (i.e., synching/merging upstream, forked repo, and local), which is explained in Step 6, before getting started on posting issues, finding solutions, and creating pull requests, which is currently in Step 5.

Actually, in addition, although perhaps it would belong to another issue, I'm not sure I understand the difference between Step 5 (GitHub Issues tutorial) and Step 8 (Create Issues and Pull Requests). Step 8 says that we will do the same as in Step 5, but now we'll create issues to improve this MDwiki for future interns, but in Step 5 we are also supposed to create issues in this repo (unless I misunderstood, of course, but I checked several times, and the only link we have in Step 5 is to this repo).

**Comment by waryhermit** [comment]
I agree with @aberdean, I ran into confusion when I tried following the steps, if you don't do step 8 before step 5 your pull request will have multiple files due to the personal MD wiki page created earlier.
